rust/feature-flags/src/api/request_handler.rs

```diff
@@ -22,10 +23,8 @@ use std::{io::Read, sync::Arc};
 #[derive(Debug, Clone, Copy, Deserialize, Serialize, PartialEq, Eq, Default)]
 #[serde(rename_all = "lowercase")]
 pub enum Compression {
-    #[serde(rename = "gzip")]
-    #[serde(alias = "gzip-js")]
+    #[serde(rename = "gzip", alias = "gzip-js")]
     Gzip,
-    Base64,
```

Comment:
we explicitly won't support b64 encoding for new flags traffic; gzip or bust babay